randovania/games/dread/exporter/options.py

```diff
@@ -12,6 +12,7 @@
 class DreadPerGameOptions(PerGameOptions):
     input_directory: Path | None = None
     target_platform: DreadModPlatform = DreadModPlatform.RYUJINX
+    linux_ryujinx_path: LinuxRyujinxPath = LinuxRyujinxPath.FLATPAK
```

Comment:
This causes old configs to get invalidated. Dunno what I need to change to fix it.

Reply:
Check Options. Somewhere there's a migration for game specific options values